pypdf/generic/_image_inline.py

```diff
@@ -0,0 +1,235 @@
+# Copyright (c) 2024, PubPub-ZZ
```

Comment:
`extract_inline_default` has been moved from `_xobj_image_helpers` to this file and thus at least the copyright from there should be copied over here as well.

I personally prefer the generic term `Copyright (c) pypdf contributors` to avoid new copyright statements for specific contributors, but this is a matter of taste. At least for now, no one really added their copyright line to each file they touched (which just complicates license compliance).